internal/util/tree/tree_test.go

```diff
@@ -279,16 +279,15 @@ func Test_V1Beta1TreePrefix(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			g := NewWithT(t)
 			var output bytes.Buffer
-
-			// Creates the output table
-			tbl := tablewriter.NewWriter(&output)
-
-			formatTableTreeV1Beta1(tbl)
+			tbl := CreateObjectTreeV1Beta1(&output)
 
 			// Add row for the root object, the cluster, and recursively for all the nodes representing the cluster status.
-			addObjectRowV1Beta1("", tbl, tt.objectTree, tt.objectTree.GetRoot())
-			tbl.Render()
-
+			err := addObjectRowV1Beta1("", tbl, tt.objectTree, tt.objectTree.GetRoot())
+			g.Expect(err).ToNot(HaveOccurred(), "Failed to add object rows")
+			if err := tbl.Render(); err != nil {
+				fmt.Printf("Error rendering table: %v", err)
+				os.Exit(1)
+			}
```

Comment:
Should this be
```suggestion
			g.Expect( tbl.Render()).To(Succeed(), "Error rendering table")
```

(same in Test_TreePrefix)